09-features.md

```diff
@@ -31,6 +31,16 @@ These flags may only be used in the `init` message:
 
 There are currently no `globalfeatures` flags.
 
+## Assigned `node_announcement` feature flags
+
+The following feature bits are currently assigned by this specification:
+
+|------|-------------------|-------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| Bits | Name              | Description                                                 | Link                                                                                                             |
+|------|-------------------|-------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| 0/1  | `var_onion_optin` | This node requires/supports variable payload routing onions | [routing onion specification](https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md) |
```

Comment:
If we want to be able to combine those with the `local` features (such as in https://github.com/lightningnetwork/lightning-rfc/pull/571) it would be better to use bits that aren't currently assigned (instead of 0/1).
I'll let you sync with @rustyrussell to see what bit values would make sense: many are tentatively assigned in https://github.com/lightningnetwork/lightning-rfc/issues/605 but I think `var_onion_optin` should replace one of those (since it's coming first). We probably need to update the list in https://github.com/lightningnetwork/lightning-rfc/issues/605.